site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -133,35 +152,35 @@ gcloud compute firewall-rules create game-server-firewall \
 
 Create a GKE cluster in which you'll install Agones. You can use
 [GKE Standard mode](#create-a-standard-mode-cluster-for-agones)
-or [GKE Autopilot mode](#create-an-autopilot-mode-cluster-for-agones)
-(Alpha).
+or [GKE Autopilot mode](#create-an-autopilot-mode-cluster-for-agones).
 
 ### Create a Standard mode cluster for Agones
 
+Create the cluster:
 ```bash
-gcloud container clusters create [CLUSTER_NAME] --cluster-version={{% gke-example-cluster-version %}} \
+gcloud container clusters create [CLUSTER_NAME] \
+  --region=[COMPUTE_REGION] \
+  --release-channel=[RELEASE_CHANNEL] \
   --tags=game-server \
   --scopes=gke-default \
   --num-nodes=4 \
-  --no-enable-autoupgrade \
   --enable-image-streaming \
   --machine-type=e2-standard-4
 ```
 
-{{< alert title="Note" color="info">}}
```

Comment:
Just to be sure, is this note being removed because it's now the default?

Reply:
Yes, exactly. It's been default since 1.22-ish (I looked it up before I removed it but can't quite remember now.)